Decisions/scs-0110-v1-ssd-flavors.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: SSD Flavors
-type: Standard
-status: Draft
+type: Decision Record
```

Comment:
Correct, this is not a full-fledged standard on its own right.
It is a decision, documenting reasoning and outcomes that then informs the flavor-naming standard.
However, since the last consolidation, both Standards and Decision Records live in the Standards directory.
The reason is that these are not always so clearly distinguishable and decisions tend to inform standards -- so keeping them together (and also have unique naming/numbering) makes it easier to keep an overview.

I have addressed this with commit https://github.com/SovereignCloudStack/standards/pull/304/commits/217d07641774f9af7af68f8b68edc967a82ee1ea